.github/workflows/wheels.yml

```
@@ -0,0 +1,67 @@
+name: Build wheels
+on:
+  push:
+    branches:
+      - master
+  create:
+    tags:
+      - v*
+jobs:
+  build:
+    name: Publish for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python_version: [3.6, 3.7, 3.8]
+        arch: [x86_64, i686]
+        os: [ubuntu-latest, macos-latest]
+        exclude:
+          - os: macos-latest
+            arch: i686
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build, install and test
+      run: |
+        if [[ "$(uname)" == "Linux" ]]; then
+          TRAVIS_OS_NAME="linux"
+        else
+          TRAVIS_OS_NAME="osx"
+        fi
+        export MB_PYTHON_OSX_VER=10.9
+        git submodule update --init --recursive
+        CONFIG_PATH=wheels/config.sh
+        pushd wheels
+          git clone https://github.com/matthew-brett/multibuild
+        popd
```

```suggestion
        (cd wheels; git clone https://github.com/matthew-brett/multibuild)
```
(i.e. using a subshell)